app/test_main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
```

Comment:
Also, make your imports according to PEP8. Blank lines, order, etc.

Reply:
done